bedrock/firefox/templates/firefox/developer/whatsnew.html

```diff
@@ -36,7 +36,7 @@
   <div class="mzp-l-content mzp-t-content-md t-center">
     <div class="mzp-c-logo mzp-t-logo-xl mzp-t-product-developer mzp-l-logo-center"></div>
     <h1>{{ ftl('firefox-developer-congrats-you-now-have-latest-v2') }}</h1>
-    {% set attrs = 'href="' ~ url('firefox.notes', channel='aurora') ~ '"' %}
+    {% set attrs = 'href="' ~ url('firefox.notes', channel='beta') ~ '"' %}
```

Comment:
Does this channel work?

```suggestion
    {% set attrs = 'href="' ~ url('firefox.notes', channel='developer') ~ '"' %}
```

Because https://www.mozilla.org/en-US/firefox/developer/notes/ would be the best of both worlds (still allow changing the destination and not hardcoding beta, but being actually the correct link nowaday.)

Or, maybe, skip the WMO redirect, and slap just a FXC link there? https://www.firefox.com/en-US/firefox/developer/notes/

Reply:
Yeah I'd use:

```suggestion
    {% set attrs = 'href="https://www.firefox.com/firefox/developer/notes/"' %}
```

and be happy about the resolution – as that's the actual correct path for this, doing the actually correct thing over at the destination;)